pkgs/by-name/ni/nix-search-cli/package.nix

```diff
@@ -17,7 +17,11 @@ buildGoModule {
 
   vendorHash = "sha256-JDOu7YdX9ztMZt0EFAMz++gD7n+Mn1VOe5g6XwrgS5M=";
 
-  passthru.updateScript = unstableGitUpdater { };
+  passthru.updateScript = unstableGitUpdater {
+    # Almost every commit is tagged as "release-<unix-time>-<commit>", software doesn't keep track of its version
+    # Using 0 feels closer to what the tagging is trying to express
```

Comment:
Similar to `treesheets`, but IMO different because it more or less just expresses the same thing as `unstable-YYYY-MM-DD`. `treesheets` actually uses its weird number internally and presents it to the user in the About dialogue, `nix-search-cli` doesn't seem to use this data.

No strong opinion on this though, I'd leave that up to the maintainer / maybe upstream themselves to decide.